OpenDreamRuntime/Objects/Types/DreamObjectImage.cs

```diff
@@ -44,17 +44,21 @@ public sealed class DreamObjectImage : DreamObject {
                     .Warning($"Attempted to create an /image from {icon}. This is invalid and a default image was created instead.");
         }
 
-        int argIndex = 1;
+        int startIndex = 0;
         DreamValue loc = args.GetArgument(1);
-        if (loc.TryGetValueAsDreamObject(out _loc)) { // If it's not a DreamObject, it's actually icon_state and not loc
-            argIndex = 2;
+        if (!loc.TryGetValueAsDreamObject(out _loc)) { // If it's not a DreamObject, it's actually icon_state and not loc
+            startIndex = 1;
+            if (!loc.IsNull) {
```

Comment:
`loc.IsNull` will always be false here because `TryGetValueAsDreamObject()` will return true for a null object.